### PR TITLE
CI: Wait cluster stabilization before running tests

### PR DESCRIPTION
**Pull request by alexandre-allard**
**Component**: ci

**Context**:
In the CI context, a lot of builds fail because we launch tests when the pods are not running or the cluster state is not stabilized yet

**Summary**:
Add a script and a step that wait for all pods to be in Running state for at least 30 seconds before continuing.

**Acceptance criteria**: 
Green build and I hope less flakies.

### CODE REVIEW

**Review by TeddyAndrieux — APPROVED** [review]
LGTM

File: eve/wait_pods_status.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            exit 0
+        fi
+        echo "All pods are not stabilized yet."
+    fi
+    echo "Waiting for pods to be $STATUS..."
+    sleep "$SLEEP_TIME"
+done
+
+echo "Pods are still not $STATUS after $RETRY retries in" \
+     "$SECONDS seconds!"
```
Comment (gdemonet) [inline eve/wait_pods_status.sh:118]: I don't see the `SECONDS` variable set anywhere :/ Maybe just:

```suggestion
     "$(( $RETRY * $SLEEP_TIME )) seconds!"
```

Comment (alexandre-allard) [reply eve/wait_pods_status.sh:118]: It's a Bash internal.

**Pull request merged**